marlowe-playground-client/src/Home.purs

```diff
@@ -0,0 +1,84 @@
+module Home where
+
+import Data.Lens (to, (^.))
+import Data.Maybe (Maybe(..))
+import Halogen (ClassName(..), ComponentHTML)
+import Halogen.Classes (blocklyIconColour, flex, fullHeight, fullWidth, haskellIcon, horizontalFlip, marloweLogo2, rightArrow, scroll, simulationIcon)
+import Halogen.HTML (a, button, div, h1, h2, h3, img, input, label, p, text)
+import Halogen.HTML.Events (onChecked, onClick)
+import Halogen.HTML.Properties (InputType(..), checked, classes, href, id_, src, target, type_)
+import Halogen.HTML.Properties as HTML
+import Prelude (const, not, (<<<))
+import Types (ChildSlots, FrontendState, HAction(..), View(..), _showHomePage)
+
+render :: forall m. FrontendState -> ComponentHTML HAction ChildSlots m
+render state =
+  div [ classes [ scroll, fullHeight ] ]
+    [ h1 [] [ text "What is the Marlowe Playground?" ]
+    , p [] [ text "For Marlowe to be usable in practice, users need to be able to understand how contracts will behave once deployed to the blockchain, but without doing the deployment. We can do that by simulating their behaviour off-chain, interactively stepping through the evaluation of a contract in the browser-based tool, the Marlowe Playground, a web tool that supports the interactive construction, revision, and simulation of smart-contracts written in Marlowe." ]
+    , h2 [] [ text "How does the playground work?" ]
+    , p [] [ text "Not sure what to put here" ]
```

Comment:
You know Halogen has underscore variants so you don't need the empty list argument. ie. you can do `h1_ [ text ...]` and so on...

Reply:
I know but I like to keep things open when I'm developing, maybe I'll clean this up then